scripts/create-new-actor.sh

```diff
@@ -151,26 +320,67 @@ struct ${actor_name}::PhaseConfig {
 void ${actor_name}::run() {
     for (auto&& config : _loop) {
         for (const auto&& _ : config) {
-            auto inserts = _totalInserts.start();
-            // TODO: main logic
+            //
+            // Evaluate the DocumentGenerator template:
+            //
             auto document = config->documentExpr();
+
+            //
+            // The clock starts running only when you call ${q}.start()${q}. The return value
+            // from this lets you record how many bytes, documents, and actual
+            // iterations (usually 1) your Actor completes while the clock is running. You then
+            // must stop the clock by calling either ${q}.success()${q} if everything is okay
+            // or ${q}.fail()${q} if there was a problem. It's undefined behavior what will
```

Comment:
Is it worth adding a small number of emojis prefixes to mark different types of comments similar to the genny ideas doc? Some way of breaking up these paragraphs might improve readability.

E.g.


👓The clock starts running only when you call ${q}.start()${q}.... # Newbie Guide of basic concepts you have to know

⚠️ It's undefined behavior what will... # Warning!

🍪 MongoException lets you include a "causing" bson document ... # Non-essential tips and tricks to improve your happiness

Reply:
I like the idea...I tried going through with it and it kinda felt contrived about what was what, though, so I just added ⚠️ wherever it seemed most appropriate.